docs/api/99_resources/locales.mdx

```diff
@@ -65,8 +65,8 @@ en:
     [2.9] usage_based_fees: Usage based fees
     [2.10] list_of_charges: List of charges used from %{from} to %{to}
     [2.11] see_breakdown: See breakdown for total unit
-    [2.12] TBD
-    [2.13] TBD
+    [2.12] true_up: True-up
+    [2.13] true_up_details: Minimum spend of {amount} prorated on days of usage
```

Comment:
Can we say "Minimum spend of {amount} prorated based on the number of days of use"?

Reply:
It's a bit long and in other language even in English, the wording will be really long (cf. DE)